docs/pages/blog/material-ui-v6-is-out.md

```diff
@@ -0,0 +1,213 @@
+<!-- a link to Next.js repo that render templates -->
+
+### React Server Components
+
+Once integrated with Pigment CSS, Material UI v6 provides a separate set of layout components, including `Grid`, `Container`, and `Stack`, that are compatible with React Server Components.
+
+```jsx
+import Box from '@mui/material-pigment-css/Box';
+import Grid from '@mui/material-pigment-css/Grid';
+import Container from '@mui/material-pigment-css/Container';
+import Stack from '@mui/material-pigment-css/Stack';
+```
+
+### Built-in sx prop support
+
+With Pigment CSS integration, all JSX elements support the `sx` prop out of the box.
+
+```diff
+- import Box from '@mui/material/Box';
+
+- <Box component="img" sx={{ padding: 2 }} />
++ <img sx={{ padding: 2 }} />
+```
+
+:::success
+Box component is **no** longer required to use the `sx` prop.
+:::
+
+## Built-in support for light and dark modes
+
+Material UI v6 supports light and dark modes out of the box.
+
+The new `colorSchemes` node lets you create a light and dark color schemes for your application:
+
+```js
+const theme = createTheme({ colorSchemes: { dark: true } }); // light is generated by default.
+
+function App() {
+  return <ThemeProvider theme={theme}>...</ThemeProvider>;
+}
+```
+
+When color schemes are used, Material UI ships these features out of the box:
+
+- CSS media `prefers-color-scheme` is used as the default method for switching between modes.
+- You can achieve instant transitions between modes with `disableTransitionOnChange` prop.
+- Selected mode is stored in the local storage to persist the user's preference.
+- The selected mode is automatically synced between browser tabs.
+
+### CSS media `prefers-color-scheme`
+
+When light and dark color schemes are enabled, Material UI v6 uses the CSS media `prefers-color-scheme` as the default method for switching between modes.
+
+This method works with server-side rendering without flickering and is compatible with all modern browsers.
+
+### A hook for toggling between modes
+
+The new `useColorScheme` hook lets you read and update the mode in your application.
+
+```jsx
+import { useColorScheme } from '@mui/material/styles';
+
+function ModeSwitcher() {
+  const { mode, setMode } = useColorScheme();
+  if (!mode) return null;
+  return (
+    <select onChange={(event) => setMode(event.target.mode)}>
+      <option value="system">System</option>
+      <option value="light">Light</option>
+      <option value="dark">Dark</option>
+    </select>
+  );
+}
+```
+
+## CSS theme variables
+
+The `cssVariables` flag converts serializable theme values like palette, spacing, typography, etc., into CSS variables.
+
+```js
+createTheme({ cssVariables: true, ... });
+```
+
+{{"component": "components/blog/material-ui-v6-is-out/ThemeTokens.js"}}
+
+With CSS variables, you can easily integrate your preferred styling solution with Material UI.
+
+```css title="styles.css"
+.custom-card {
+  background-color: var(--palette-background-default);
```

Comment:
Is it a type that the "mui" is not present in these variables. It is a bit conofusing as you read it, as in the previous section all variables start with the mui prefix.

Reply:
Nice catch.